### ISSUE TITLE
Duplicates forming after dragging

### ISSUE THREAD

**Issue by bikingbadger** [SortableJS/vue.draggable.next#28]
I'm trying to get vue.draggable to work and overall it's great. There is an issue that sometimes I drag an item and it moves into position in the background but it's duplicated in the list shown on my app. After refreshing the page it actually shows the correct result. Any help trying to debug this would be great.

### Example link
https://github.com/bikingbadger/pompom/blob/develop/src/components/tasks/TaskList.vue

### Step by step scenario
I  drag an item into a new position and it moves but not to the correct place but instead creates a duplicate of another item.

### Expected Solution
Should drag and not create duplicates

**Comment by bikingbadger** [comment]
Anyone know how I could try and debug this issue?

**Comment by veraangerer** [comment]
Having the same problem but for me it seems to happen only if I move an item inside a list and it happens that it protrudes over the edge of the list accidentally. It then duplicates the item in the list (but only for the moment, the state still stays untouched and reflects the current state as it should be.

**Comment by bikingbadger** [comment]
It's inconsistent for me but what I see is that it creates a duplicate on the ui but in the background the positioning is still using the state so when I then start to drag and drop the actual state item is moved even though I'm dragging the duplicate.

**Comment by veraangerer** [comment]
Okay thats really weird. What I could observe is that if I have an item in my list twice already (on purpose) the duplication happened when I started dragging & dropping but with every item being in the list only once this did not happen.

**Comment by bikingbadger** [comment]
This seems to be an actual Vue 3 issue. I have created my own drag and drop and also getting the issue.

**Comment by buianh98** [comment]
You can use id-key as index instead of id, problem will be solved.
![vuedraggable](https://user-images.githubusercontent.com/76531650/119523916-5f3db600-bda7-11eb-8c80-3fd127b3ebae.png)

**Comment by David-Desmaisons** [comment]
@bikingbadger What is the template you are using? To me it seems that you are using different items with the same id which causes the inconsistency.
